<!doctype html>
<html lang="en"><head><meta charSet="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><meta name="viewport" content="width=device-width, initial-scale=1"/><title>Test Report</title><link rel="stylesheet" href="assets/app.css"/></head><body data-raw="{&quot;stats&quot;:{&quot;suites&quot;:2,&quot;tests&quot;:27,&quot;passes&quot;:0,&quot;pending&quot;:0,&quot;failures&quot;:27,&quot;start&quot;:&quot;2022-03-15T16:25:27.669Z&quot;,&quot;end&quot;:&quot;2022-03-15T16:25:27.839Z&quot;,&quot;duration&quot;:170,&quot;testsRegistered&quot;:27,&quot;passPercent&quot;:0,&quot;pendingPercent&quot;:0,&quot;other&quot;:0,&quot;hasOther&quot;:false,&quot;skipped&quot;:0,&quot;hasSkipped&quot;:false},&quot;results&quot;:[{&quot;uuid&quot;:&quot;d77a6182-2044-413e-a888-b53b6bad4da3&quot;,&quot;title&quot;:&quot;&quot;,&quot;fullFile&quot;:&quot;&quot;,&quot;file&quot;:&quot;&quot;,&quot;beforeHooks&quot;:[],&quot;afterHooks&quot;:[],&quot;tests&quot;:[],&quot;suites&quot;:[{&quot;uuid&quot;:&quot;a0f3e7f3-5aee-48e0-a95e-afdc613d3ac2&quot;,&quot;title&quot;:&quot;Threshold time testing&quot;,&quot;fullFile&quot;:&quot;&quot;,&quot;file&quot;:&quot;&quot;,&quot;beforeHooks&quot;:[],&quot;afterHooks&quot;:[],&quot;tests&quot;:[],&quot;suites&quot;:[{&quot;uuid&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;title&quot;:&quot; First Class of Feedback&quot;,&quot;fullFile&quot;:&quot;&quot;,&quot;file&quot;:&quot;&quot;,&quot;beforeHooks&quot;:[],&quot;afterHooks&quot;:[],&quot;tests&quot;:[{&quot;title&quot;:&quot;Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.758700000355 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.758700000355 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.758700000355\n+ 1\n&quot;},&quot;uuid&quot;:&quot;e31bea38-2fb0-46bd-985a-e33640a1d74c&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.76900000008754 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.76900000008754 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.76900000008754\n+ 1\n&quot;},&quot;uuid&quot;:&quot;f014933f-421d-482c-9faf-6ca96cc79006&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:3,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.5596000000369 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.5596000000369 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.5596000000369\n+ 1\n&quot;},&quot;uuid&quot;:&quot;d12794d4-bc4e-4b79-8c28-c717e057bde3&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:3,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.60150000010617 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.60150000010617 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.60150000010617\n+ 1\n&quot;},&quot;uuid&quot;:&quot;fe62c242-f979-46b9-b1c3-0fc88ef40e2e&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:8,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 314.53770000021905 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 314.53770000021905 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 314.53770000021905\n+ 100\n&quot;},&quot;uuid&quot;:&quot;09c72826-21e1-4dbb-af77-b96de49306bc&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.46320000011474 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.46320000011474 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.46320000011474\n+ 100\n&quot;},&quot;uuid&quot;:&quot;b9fab8b8-72d9-4ea6-bd2e-967197d82be6&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:7,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.9161000002641 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.9161000002641 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.9161000002641\n+ 100\n&quot;},&quot;uuid&quot;:&quot;8c77ca06-85e1-4184-868c-544127128d46&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:4,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.5053000000771 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.5053000000771 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.5053000000771\n+ 100\n&quot;},&quot;uuid&quot;:&quot;14098e5a-ded4-4017-b511-54df7fd0b0b8&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:3,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.6369000002742 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.6369000002742 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.6369000002742\n+ 1\n&quot;},&quot;uuid&quot;:&quot;f374be65-12fc-4044-9355-188377e40720&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:4,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.9165000002831 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.9165000002831 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.9165000002831\n+ 1\n&quot;},&quot;uuid&quot;:&quot;d02667c0-bf16-408d-8c9b-d58b24fa61bd&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.4432000000961 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.4432000000961 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.4432000000961\n+ 1\n&quot;},&quot;uuid&quot;:&quot;ec795dbb-a572-4c5c-9e8f-94f8af148922&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:4,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.6427000002004 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.6427000002004 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.6427000002004\n+ 1\n&quot;},&quot;uuid&quot;:&quot;3b77a58f-0d9f-4700-a446-965746742162&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:3,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.9790000002831 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.9790000002831 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.9790000002831\n+ 1\n&quot;},&quot;uuid&quot;:&quot;ece1d5fb-7a0d-47c2-9297-225f717e261a&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:8,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 314.1107000003103 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 314.1107000003103 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 314.1107000003103\n+ 100\n&quot;},&quot;uuid&quot;:&quot;57270d53-d51b-4140-930f-80e51e750330&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.21490000002086 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.21490000002086 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.21490000002086\n+ 100\n&quot;},&quot;uuid&quot;:&quot;73bad2ae-b0bd-4908-b8df-26346cd560a4&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:9,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 315.18980000005104 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 315.18980000005104 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 315.18980000005104\n+ 100\n&quot;},&quot;uuid&quot;:&quot;bbdc31d5-1564-4c4b-9709-fa1371f36af3&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.17310000001453 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.17310000001453 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.17310000001453\n+ 100\n&quot;},&quot;uuid&quot;:&quot;84f7f36a-86f8-4b56-9adf-ddc5f0de82c5&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:9,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 314.7369000001345 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 314.7369000001345 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 314.7369000001345\n+ 1\n&quot;},&quot;uuid&quot;:&quot;f2e7284d-4e36-4baa-a97a-afcb4cf04bb4&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;alreadyDidItBefore\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:4,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.82230000011623 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.82230000011623 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.82230000011623\n+ 1\n&quot;},&quot;uuid&quot;:&quot;9b02cc41-0365-4659-89fc-732166fe7eab&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;frequemcy\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:3,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 309.7585000002291 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 309.7585000002291 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 309.7585000002291\n+ 1\n&quot;},&quot;uuid&quot;:&quot;12546961-1784-4437-961d-96b17f013520&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;hint\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:8,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 313.974100000225 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 313.974100000225 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 313.974100000225\n+ 1\n&quot;},&quot;uuid&quot;:&quot;667072ac-4e62-4685-b1e5-b806c4d53371&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;inputTestCase\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:4,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.90740000014193 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.90740000014193 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.90740000014193\n+ 1\n&quot;},&quot;uuid&quot;:&quot;c61cb081-8973-4148-a52c-f1217f1c9b46&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivate\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 310.51370000001043 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 310.51370000001043 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 310.51370000001043\n+ 100\n&quot;},&quot;uuid&quot;:&quot;73de11bd-35d7-4807-99a1-75e4c36da606&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;motivateHarder\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.81099999998696 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.81099999998696 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.81099999998696\n+ 100\n&quot;},&quot;uuid&quot;:&quot;fe88ff69-8791-4292-8861-45be9d11e5d7&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;outputTestCase\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:5,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.5352000000421 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.5352000000421 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.5352000000421\n+ 100\n&quot;},&quot;uuid&quot;:&quot;53896221-9553-437f-b5fe-f8f253604c25&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;percentage\&quot; finish earlier  than 100 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:6,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 312.3889999999665 to be below 100&quot;,&quot;estack&quot;:&quot;AssertionError: expected 312.3889999999665 to be below 100\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 312.3889999999665\n+ 100\n&quot;},&quot;uuid&quot;:&quot;92887968-0abc-4392-9f70-486c39c6ba83&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false},{&quot;title&quot;:&quot;Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;fullTitle&quot;:&quot;Threshold time testing  First Class of Feedback Validate if feedback \&quot;submissionNumbers\&quot; finish earlier  than 1 ms&quot;,&quot;timedOut&quot;:false,&quot;duration&quot;:6,&quot;state&quot;:&quot;failed&quot;,&quot;pass&quot;:false,&quot;fail&quot;:true,&quot;pending&quot;:false,&quot;code&quot;:&quot;var self = this, args = arguments; return new Promise(function (resolve, reject) { var gen = fn.apply(self, args); function _next(value) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;next\&quot;, value); } function _throw(err) { asyncGeneratorStep(gen, resolve, reject, _next, _throw, \&quot;throw\&quot;, err); } _next(undefined); });&quot;,&quot;err&quot;:{&quot;message&quot;:&quot;AssertionError: expected 311.99840000015683 to be below 1&quot;,&quot;estack&quot;:&quot;AssertionError: expected 311.99840000015683 to be below 1\n    at _callee2$ (dist-server/tests/sample-tests.js:155:47)\n    at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)\n    at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)\n    at asyncGeneratorStep (dist-server/tests/sample-tests.js:25:103)\n    at _next (dist-server/tests/sample-tests.js:27:194)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)&quot;,&quot;diff&quot;:&quot;- 311.99840000015683\n+ 1\n&quot;},&quot;uuid&quot;:&quot;4d574a06-5952-4d64-8b8d-820d8ad15781&quot;,&quot;parentUUID&quot;:&quot;141f66ab-545e-4104-8f37-a69cbd3ac7de&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false}],&quot;suites&quot;:[],&quot;passes&quot;:[],&quot;failures&quot;:[&quot;e31bea38-2fb0-46bd-985a-e33640a1d74c&quot;,&quot;f014933f-421d-482c-9faf-6ca96cc79006&quot;,&quot;d12794d4-bc4e-4b79-8c28-c717e057bde3&quot;,&quot;fe62c242-f979-46b9-b1c3-0fc88ef40e2e&quot;,&quot;09c72826-21e1-4dbb-af77-b96de49306bc&quot;,&quot;b9fab8b8-72d9-4ea6-bd2e-967197d82be6&quot;,&quot;8c77ca06-85e1-4184-868c-544127128d46&quot;,&quot;14098e5a-ded4-4017-b511-54df7fd0b0b8&quot;,&quot;f374be65-12fc-4044-9355-188377e40720&quot;,&quot;d02667c0-bf16-408d-8c9b-d58b24fa61bd&quot;,&quot;ec795dbb-a572-4c5c-9e8f-94f8af148922&quot;,&quot;3b77a58f-0d9f-4700-a446-965746742162&quot;,&quot;ece1d5fb-7a0d-47c2-9297-225f717e261a&quot;,&quot;57270d53-d51b-4140-930f-80e51e750330&quot;,&quot;73bad2ae-b0bd-4908-b8df-26346cd560a4&quot;,&quot;bbdc31d5-1564-4c4b-9709-fa1371f36af3&quot;,&quot;84f7f36a-86f8-4b56-9adf-ddc5f0de82c5&quot;,&quot;f2e7284d-4e36-4baa-a97a-afcb4cf04bb4&quot;,&quot;9b02cc41-0365-4659-89fc-732166fe7eab&quot;,&quot;12546961-1784-4437-961d-96b17f013520&quot;,&quot;667072ac-4e62-4685-b1e5-b806c4d53371&quot;,&quot;c61cb081-8973-4148-a52c-f1217f1c9b46&quot;,&quot;73de11bd-35d7-4807-99a1-75e4c36da606&quot;,&quot;fe88ff69-8791-4292-8861-45be9d11e5d7&quot;,&quot;53896221-9553-437f-b5fe-f8f253604c25&quot;,&quot;92887968-0abc-4392-9f70-486c39c6ba83&quot;,&quot;4d574a06-5952-4d64-8b8d-820d8ad15781&quot;],&quot;pending&quot;:[],&quot;skipped&quot;:[],&quot;duration&quot;:141,&quot;root&quot;:false,&quot;rootEmpty&quot;:false,&quot;_timeout&quot;:200000}],&quot;passes&quot;:[],&quot;failures&quot;:[],&quot;pending&quot;:[],&quot;skipped&quot;:[],&quot;duration&quot;:0,&quot;root&quot;:false,&quot;rootEmpty&quot;:false,&quot;_timeout&quot;:200000}],&quot;passes&quot;:[],&quot;failures&quot;:[],&quot;pending&quot;:[],&quot;skipped&quot;:[],&quot;duration&quot;:0,&quot;root&quot;:true,&quot;rootEmpty&quot;:true,&quot;_timeout&quot;:200000}],&quot;meta&quot;:{&quot;mocha&quot;:{&quot;version&quot;:&quot;9.2.1&quot;},&quot;mochawesome&quot;:{&quot;options&quot;:{&quot;quiet&quot;:false,&quot;reportFilename&quot;:&quot;mochawesome&quot;,&quot;saveHtml&quot;:true,&quot;saveJson&quot;:true,&quot;consoleReporter&quot;:&quot;spec&quot;,&quot;useInlineDiffs&quot;:false,&quot;code&quot;:true},&quot;version&quot;:&quot;7.0.1&quot;},&quot;marge&quot;:{&quot;options&quot;:{&quot;reportDir&quot;:&quot;/mnt/e/INESC TEC/feedback-manager/public/execution-report/2022/03/15/16/25/26&quot;,&quot;reportTitle&quot;:&quot;Test Execution Report&quot;,&quot;reportPageTitle&quot;:&quot;Test Report&quot;,&quot;charts&quot;:true},&quot;version&quot;:&quot;6.0.1&quot;}}}" data-config="{&quot;reportFilename&quot;:&quot;mochawesome&quot;,&quot;reportDir&quot;:&quot;/mnt/e/INESC TEC/feedback-manager/public/execution-report/2022/03/15/16/25/26&quot;,&quot;reportTitle&quot;:&quot;Test Execution Report&quot;,&quot;reportPageTitle&quot;:&quot;Test Report&quot;,&quot;inline&quot;:false,&quot;inlineAssets&quot;:false,&quot;cdn&quot;:false,&quot;charts&quot;:true,&quot;enableCharts&quot;:true,&quot;code&quot;:true,&quot;enableCode&quot;:true,&quot;autoOpen&quot;:false,&quot;overwrite&quot;:true,&quot;timestamp&quot;:false,&quot;ts&quot;:false,&quot;showPassed&quot;:true,&quot;showFailed&quot;:true,&quot;showPending&quot;:true,&quot;showSkipped&quot;:false,&quot;showHooks&quot;:&quot;failed&quot;,&quot;saveJson&quot;:true,&quot;saveHtml&quot;:true,&quot;dev&quot;:false,&quot;assetsDir&quot;:&quot;/mnt/e/INESC TEC/feedback-manager/public/execution-report/2022/03/15/16/25/26/assets&quot;,&quot;jsonFile&quot;:&quot;/mnt/e/INESC TEC/feedback-manager/public/execution-report/2022/03/15/16/25/26/mochawesome.json&quot;,&quot;htmlFile&quot;:&quot;/mnt/e/INESC TEC/feedback-manager/public/execution-report/2022/03/15/16/25/26/mochawesome.html&quot;}"><div id="report"></div><script src="assets/app.js"></script></body></html>